Formula/simple-obfs.rb

```diff
@@ -0,0 +1,23 @@
+class SimpleObfs < Formula
+  desc "Simple obfusacting plugin of shadowsocks-libev."
+  homepage "https://github.com/shadowsocks/simple-obfs"
+  url "https://github.com/shadowsocks/simple-obfs/archive/v0.0.2.tar.gz"
+  sha256 "cfd7b847be57401e1c664662781d41bc5b089b341232d94375ca4dd612b3852f"
+
+  depends_on "asciidoc" => :build
+  depends_on "xmlto" => :build
+  depends_on "pcre"
+  depends_on "openssl"
+
+  def install
+    ENV["XML_CATALOG_FILES"] = etc/"xml/catalog"
+
+    system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    assert_match "ERROR: Invalid config path.", shell_output("#{bin}/obfs-local -c test.json 2>&1", 1)
```

Comment:
It means lack of a config file, not running with error.
I write the test based on [this formula ](https://github.com/Homebrew/homebrew-core/blob/11249072d68d7883258a68ff10601daf4e4b5210/Formula/proxychains-ng.rb).

Of course I want to test with config file, like [this formula](https://github.com/Homebrew/homebrew-core/blob/37e39a44e51191a22678376922c5a8ca3ccc240d/Formula/shadowsocks-libev.rb). But actually this is just a plugin for it. It may not work right independently.